Too bad, our rebase has other arguments, and they're not trivial to translate.

rebase -C ../wntgcci.pro/misc/coffbase.txt -b 0x68000000 -e 10000 -l ../wntgcci.pro/misc/rebase_log.txt -R /home/janneke/vc/gub/target/mingw/build/openoffice-trunk/build/ooo300-m9/solver/300/wntgcci.pro/bin -N no_rebase.txt -d -v @../wntgcci.pro/misc/rebase_new.txt
/home/janneke/vc/gub/target/tools/root/usr/bin/rebase: unknown option -- C
usage: rebase -b BaseAddress [-Vdv] [-o Offset] [-T FileList | -] Files...
Error: rebase failed with exit code 1!


--- postprocess/rebase/makefile.mk.~1.10.~	2008-04-10 10:48:57.000000000 +0200
+++ postprocess/rebase/makefile.mk	2008-11-07 17:46:19.000000000 +0100
@@ -48,13 +48,13 @@ IMAGENAMES=$(SOLARBINDIR)$/*.dll $(SOLAR
 ALLTAR : REBASE
 
 REBASE .PHONY : $(BASEADDRESSES)
-.IF "$(GUI)"=="WNT"
+.IF "$(GUI)"=="WNT" && "$(CROSS_COMPILING)" == "no"
 .IF "$(product)"=="full"
 	$(PERL) rebase.pl -C $(BASEADDRESSES) -b $(STARTADDRESS) -d -e 10000 -l $(LOGFILE) -m $(MISC) -v -R $(SOLARBINDIR) -N $(EXCLUDELIST) $(IMAGENAMES)
 .ELSE	# "$(product)"=="full"
 	@echo Doing nothing on non product builds ...
 .ENDIF	# "$(product)"=="full"
-.ELSE	# "$(GUI)"=="WNT"
+.ELSE	# "$(GUI)"!="WNT" || "$(CROSS_COMPILING)" != "no"
 	@echo Nothing to do, 'rebase' is windows only.
 .ENDIF
 
